Makefile

```diff
@@ -33,8 +33,7 @@ endif
 
 GO_LDFLAGS = -ldflags "$(STATIC_LINKING_LDFLAGS) \
 						 -X github.com/dividat/driver/src/dividat-driver/server.channel=$(CHANNEL) \
```

Comment:
Because we now only have main, we could remove this information that seems to be used only in the logs.

Reply:
Fair enough: fe6405b